docs/content/mobile/android/local-deploy.md

```diff
@@ -34,7 +40,7 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
 
 1. You should now see your Unity Editor's game view mirrored on your Android device.
 
-## Use your Android emulator {#emulator}
+## Connecting to a local deployment using Android Emulator{#android-emulator}
```

Comment:
cc @ElleEss as she was the one that made the change to "Use ..." in the first place, I believe

Reply:
Changed Connecting to Connect to clarify that the heading is a call to action. Reached out to @ElleEss to check her thoughts.

Reply:
Confirmed with @ElleEss that this is now fine as it's not a gerund.

In other news today I learned what gerunds are.